instrumentation/jedis/jedis-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jedis/v3_0/JedisMethodMatch.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jedis.v3_0;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+public class JedisMethodMatch {
+
+  public static ElementMatcher<MethodDescription> getJedisMethodMatcher() {
```

Comment:
If the goal is to get all methods that correspond to jedis commands then could also experiment with something like
```
        isMethod()
            .and(isPublic())
            .and(hasSuperMethod(methodIsDeclaredByType(nameEndsWith("Commands"))))
```

Reply:
> If the goal is to get all methods that correspond to jedis commands then could also experiment with something like
> 
> ```
>         isMethod()
>             .and(isPublic())
>             .and(hasSuperMethod(methodIsDeclaredByType(nameEndsWith("Commands"))))
> ```

Hi, @laurit I used the code above and it seems not working with `slaveOfNoOne` and `select` methods that inherited from `BinaryJedis `.